services/src/test/java/org/keycloak/protocol/oid4vc/OID4VCClientRegistrationProviderTest.java

```diff
@@ -0,0 +1,115 @@
+package org.keycloak.protocol.oid4vc;
```

Comment:
Could you add header comments the same as other newly created files?

Reply:
Fixed in  [7beca5e](https://github.com/keycloak/keycloak/pull/27931/commits/7beca5e54dc584c6726a73ca82109f0fc3d22124)

Reply:
Thank you.